docker-compose.yml

```
@@ -14,7 +14,7 @@ services:
         condition: service_healthy
     environment:
       - PGPASSWORD=${credentials_PGPASSWORD}
-    image: ubuntu:kinetic
+    image: ubuntu:latest
```

```suggestion
    image: ubuntu:jammy
```

It's better to use the LTS release